install.sh

```diff
@@ -1,5 +1,9 @@
 #!/usr/bin/env bash
 
+# SPDX-FileCopyrightText: 2019 Adafruit for Adafruit Industries
```

Comment:
Just make this "Adafruit Industries", since "Adafruit" is not a person.

Reply:
Updated here, and in the LICENSE file (where I copied this from).